src/openforms/emails/templates/emails/templatetags/appointment_information.html

```diff
@@ -53,17 +53,12 @@ <h2 style="margin: 1em 0; font-size: 1.3em;">{% trans "Your contact details" %}<
     </table>
 
     <p>
-        {% trans "If you want to cancel or change your appointment, you can do so below." %}
+        {% trans "If you want to cancel your appointment, you can do so below." %}
     </p>
     <p>
-        <a href="{{ appointment_cancel_link }}" rel="nofollow">{% trans "Cancel appointment" %}</a>
+        {% trans "In case you want to modify it you have to cancel it and then create a new one." %}
```

Comment:
I think the previous message was a bit nicer/more descriptive.

```suggestion
        {% trans "If you wish to change your appointment, please cancel it using the link below and create a new one." %}
```